packages/react-native-bpk-component-text/stories.js

```diff
@@ -109,28 +109,28 @@ storiesOf('react-native-bpk-component-text', module)
   ))
   .add('Colours', () => (
     <View>
-      <BpkText textStyle="xxxl" style={{ color: colorBlue700 }}>
+      <BpkText textStyle="xxxl" style={{ color: colorSkyBlue }}>
```

Comment:
We should check if this blue should be Sky Blue and not another Blue

Reply:
this is just stories in the example app, I don't think it really matters

Reply:
That is true!